crates/core_arch/src/x86_64/mod.rs

```diff
@@ -3,6 +3,40 @@
 #[macro_use]
 mod macros;
 
+#[unstable(feature = "x86_amx_intrinsics", issue = "126622")]
+#[allow(non_camel_case_types)]
+#[repr(packed)]
+pub struct __tilecfg {
+    /// 0 `or` 1
+    pub palette: u8,
+    pub start_row: u8,
+    /// reserved, must be zero
+    reserved_a0: [u8; 14],
+    /// number of bytes of one row in each tile
+    pub colsb: [u16; 8],
+    /// reserved, must be zero
+    reserved_b0: [u16; 8],
+    /// number of rows in each tile
+    pub rows: [u8; 8],
+    /// reserved, must be zero
+    reserved_c0: [u8; 8],
+}
+
+#[unstable(feature = "x86_amx_intrinsics", issue = "126622")]
+impl Default for __tilecfg {
+    fn default() -> Self {
+        __tilecfg {
+            palette: 0,
+            start_row: 0,
+            reserved_a0: [0; 14],
+            colsb: [0; 8],
+            reserved_b0: [0; 8],
+            rows: [0; 8],
+            reserved_c0: [0; 8],
+        }
+    }
+}
```

Comment:
Can't this be `Derive`d?

Reply:
Thx for the advice, implement several basic traits for __tilecfg via `derive`